db/migrate/20191023180308_subclass_ems_cluster.rb

```diff
@@ -0,0 +1,45 @@
+class SubclassEmsCluster < ActiveRecord::Migration[5.1]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class EmsCluster < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+  	# Firstly, rename the old OpenStack subclassing
+  	EmsCluster.where(type: 'ManageIQ::Providers::Openstack::InfraManager::EmsCluster'}).
```

Comment:
These should be scoped by `.in_my_region`

Reply:
@agrare I based this PR on https://github.com/ManageIQ/manageiq-schema/blob/master/db/migrate/20151125081618_set_correct_sti_type_on_openstack_infra_miq_template.rb where `.in_my_region` was not used.